docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+### 1.2.1 Prepare Datset for Yarn-Cluster
+First you should download the Fashion-MNIST dataset manually on your local machine.
+
+```bash
+# PyTorch official dataset download link
+git clone https://github.com/zalandoresearch/fashion-mnist.git
+mv /path/to/fashion-mnist/data/fashion /path/to/local/data/FashionMNIST/raw 
+```
+
+After downloading the dataset, you need to upload it to a network file system (HDFS or S3).
+
+```bash
+hdfs dfs -put /path/to/local/data/FashionMNIST hdfs://path/to/remote/data
+```
+
+Please set the dataset path on the network file system through `--remote-dir` argument when submmiting and running the program on yarn-cluster, the dataset would be downloaded and processing on the driver.
+
+```bash
+python train.py --cluster_mode yarn-cluster --remote-dir hdfs://path/to/remote/data
+```
+
+# 2. Prepare Environment
+Before submitting the BigDL program to Yarn Cluster, we need to setup the environment as the following steps:
+* Setup JAVA & Hadoop Environment
+* Install All Needed Python Libraries 
+* For CDH Users
+
+## 2.1 Setup JAVA & Hadoop Environment
+We need to download and install JDK in the environment, and properly set the environment variable JAVA_HOME, which is required by Spark. JDK8 is highly recommended.
+
+### 2.1.1 Setup JAVA Environment
+```bash
+# For Ubuntu
+sudo apt-get install openjdk-8-jre
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
+
+# For CentOS
+su -c "yum install java-1.8.0-openjdk"
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/jre
+
+export PATH=$PATH:$JAVA_HOME/bin
+java -version  # Verify the version of JDK.
+```
+
+### 2.1.2 Setup Hadoop Environment
+Check the Hadoop setup and configurations of our cluster. Make sure we correctly set the environment variable `HADOOP_CONF_DIR`, which is needed to initialize Spark on YARN:
+```bash
+export HADOOP_CONF_DIR=/path/to/hadoop/conf
+```
+
+## 2.2 Install Needed Python Libraries
+### 2.2.1 Use conda to install python libraries
+We need first to use conda to prepare the Python environment on the local machine where we submit our application. Create a conda environment, install BigDL and all the needed Python libraries in the created conda environment:
+``` bash
+conda create -n bigdl python=3.7  # "bigdl" is conda environment name, you can use any name you like.
+conda activate bigdl
+```
+Then install BigDL and needed Python libraries in the Conda environment.
+```bash
+# Use conda or pip to install all the needed Python dependencies in the created conda environment.
+pip install bigdl
+```
+Installing BigDL will automatically install all the BigDL packages including bigdl-nano, bigdl-dllib, bigdl-orca, bigdl-chronos, bigdl-friesian, bigdl-serving and their dependencies if they haven’t been detected in your conda environment.
```

Comment:
no need to discuss bigdl-nano, bigdl-dllib, etc. here; instead, talk about that it will automatically install PySpark, and the options to support spark2 and spark3